src/components/dropdown/Dropdown.tsx

```diff
@@ -35,7 +35,8 @@ export function Dropdown(props: Props) {
         interactive={true}
         render={() => (
           <div
-            className={`box w-48 rounded-md shadow-lg py-1 bg-white dark:bg-gray-800 ring-1 ring-black ring-opacity-5 focus:outline-none ${props.className}`}
+            className={`box rounded-md shadow-lg py-1 bg-white dark:bg-gray-800 ring-1 ring-black ring-opacity-5 focus:outline-none ${props.className}`}
+            style={{ minWidth: 192, maxWidth: 235 }}
           >
```

Comment:
The reason for the style change for this component is the wider width of all the dropdown menus when we included the icons at the beginning. Tailwind doesn't give us a `min-w-48` (for example) class to set, so I had to set it through the style property.

Reply:
This is okay, but Tailwind min/max are defined in REMs. Can we do the same? Like this (I think) it's in pixels which means it's not fully reactive to different screen sizes?

Reply:
Changed